unitex/src/fr/umlv/unitex/frames/GraphPathDialog.java

```diff
@@ -180,10 +206,12 @@ public void actionPerformed(ActionEvent arg0) {
 				fst2 = new File(FileUtil.getFileNameWithoutExtension(graphName
```

Comment:
I added code to UnitexFrame.java to display error message if user clicks explore paths menu option before a graph has not been created